### DISCUSSION TITLE
How do i make profile specifc config

### DISCUSSION THREAD

**Discussion by OmiChrome** [hooke007/mpv_PlayKit#288] [z_ENG_discuss]
the issue is extents of https://github.com/hooke007/MPV_lazy/discussions/285
or a little workaround for that 
```
[extension.webm]
profile=image # append already ready profile for images
--input-conf=PATH
```
i simply made some profile , 
but `--input-conf` wasn't working for that part 
**problem **
all options show error 
```
The config-dir option can't be used in a config file.
Error parsing option config-dir (option parameter could not be parsed)
mpv-git/portable_config/profiles.conf:23: setting option `config-dir='~~/mvi/' failed.`
```
i even tried `input-conf` 
but still no clue ?

**what i want?**
use file-specifc or profile specific `config-dir`

**Comment by OmiChrome** [comment]
tldr:  make config based on file-types or [profile]

**Comment by hooke007** [comment]
You cannot do the thing that mpv doesn't allow you to do.

**Reply by OmiChrome** [reply]
😭  i was thinking mpv was so good
